src/Constants.h

```diff
@@ -69,6 +69,10 @@
 // Shared region polygon approximation
 #define DEFAULT_VERTEX_COUNT 1000
 
+// file list
+#define FILE_LIST_FIRST_PROGRESS_AFTER_SECS 5
+#define UPDATE_FILE_LIST_PROGRESS_PER_SECS 2
```

Comment:
How about `FILE_LIST_PROGRESS_INTERVAL_SECS`, similar to what you did for the image moments constant in the other PR?